students/eldonab/lesson09/mailroom_oo/donor_models.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+
+class Donor():
+    """create a class that represents a donor."""
+    def __init__(self, name, donations = None):
+        self.name = name
+        self.donations = donations
+
+
+    def add_donation(self, amount):
```

Comment:
note that module functions require two lines space, but Class methods require one line space between them.